tests/appsec/iast/test_iast.py

```diff
@@ -26,9 +26,6 @@ class Test_Iast(BaseTestCase):
     else:
         EXPECTED_LOCATION = ""  # (TBD)
 
-    @missing_feature(
```

Comment:
Is this test case was ok on version `0.108.0`? if no, you must keep the line, replacing `library="java"` by `context.library<"java@x.y.z`, where `x.y.z` is the version that fix this test case

Reply:
Thanks for the info! Since we don't know exactly in which version it will be released I've reverted the change and we will come back to it once it's done.